yarn/src/test/scala/org/apache/spark/scheduler/cluster/StubApplicationAttemptId.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
I'm using them more in the tests in the later patches. I can (and will) move them into the test helper, but be assured, there's a lot more tests to come.